internal/slicer/slicer.go

```diff
@@ -284,6 +298,13 @@ func Run(options *RunOptions) (*Report, error) {
 		RootDir:    targetDirAbs,
 		CheckWrite: checkWrite,
 		CheckRead:  checkRead,
+		Create: func(opts *fsutil.CreateOptions) error {
+			entry, err := fsutil.Create(opts)
+			if err != nil {
+				return err
+			}
+			return report.Mutate(entry)
+		},
```

Comment:
I think the function here should be defined above following the convention of `checkRead` and `checkWrite`.